### PR TITLE
Update busybox

### PR DESCRIPTION
**Pull request by tianon**
Changes:

- https://github.com/docker-library/busybox/commit/6ded0a4: Merge pull request https://github.com/docker-library/busybox/pull/188 from infosiftr/reproducible-rootfs
- https://github.com/docker-library/busybox/commit/7e39d61: Initial `Builder: oci-import` support
- https://github.com/docker-library/busybox/commit/644ee1a: Adjust tarball creation to be reproducible

### CODE REVIEW

**Comment by tianon** [comment]
Doh, and of course CI is running into https://github.com/docker-library/bashbrew/pull/93 so I'll need to do a Bashbrew release too. :+1:

**Comment by tianon** [comment]
(The bot will probably clobber that, but I wanted to test that the release actually does fix the errors.)

File: library/busybox [hunk 1]
```diff
@@ -1,86 +1,130 @@
...
-s390x-Directory: latest/glibc
+Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, riscv64, s390x, arm32v6
+amd64-Directory: latest/glibc/amd64
+arm32v5-Directory: latest/glibc/arm32v5
+arm32v7-Directory: latest/glibc/arm32v7
+arm64v8-Directory: latest/glibc/arm64v8
+i386-Directory: latest/glibc/i386
+mips64le-Directory: latest/glibc/mips64le
+ppc64le-Directory: latest/glibc/ppc64le
+riscv64-Directory: latest/glibc/riscv64
```
[the remark this answers is not in the document]
Comment (tianon) [reply library/busybox:82]: https://github.com/docker-library/busybox/commit/0fd3015624742ae0c2afdab536129d1ed2e07fc4 :+1:

**Comment by tianon** [comment]
Ok, actual bashbrew update in https://github.com/docker-library/official-images/pull/16361

**Comment by tianon** [comment]
This will be our first real, official test of https://github.com/docker-library/meta-scripts/pull/20 :eyes:

**Comment by tianon** [comment]
Just to be extra cute, here's a `diffoscope` of "old uclibc" to "new uclibc" build for `amd64`:

```console
$ wget -O busybox-a.tar.xz 'https://github.com/docker-library/busybox/raw/09ee80aedec1d8c604f104e8bec41ed19274620a/latest/uclibc/busybox.tar.xz'
...
$ wget -O busybox-b.tar.gz 'https://github.com/docker-library/busybox/raw/d0b7d566eb4f1fa9933984e6fc04ab11f08f4592/latest/uclibc/amd64/rootfs.tar.gz'
...
$ sha256sum busybox-*
b9c12f14c9ab4e0652a809e4a5b1d9ef65ec3f79fca0478151e3fb9fc9d4b0d7  busybox-a.tar.xz
cd82127e91e4b4656e681824cb3687b64060f91d9b44a695055a3475ac0dcd89  busybox-b.tar.gz
```

<details>
<summary>Diff:</summary>

```diff
$ docker run -it --rm --mount type=bind,src="$PWD",dst=/wd,ro --workdir /wd tianon/diffoscope diffoscope --exclude-directory-metadata recursive busybox-*
--- busybox-a.tar.xz
+++ busybox-b.tar.gz
├── filetype from file(1)
│ @@ -1 +1 @@
│ -XZ compressed data
│ +gzip compressed data, from Unix
├── filetype from diffoscope
│ @@ -1 +1 @@
│ -XzFile
│ +GzipFile
│   --- busybox-a.tar
├── +++ busybox-b.tar
│ ├── file list
│ │ @@ -1,429 +1,429 @@
│ │  ./
│ │ -./bin/
│ │ -./bin/mv
│ │ -./bin/iptunnel
│ │ -./bin/xzcat
│ │ -./bin/runsv
│ │ -./bin/fstrim
│ │ -./bin/mount
│ │ -./bin/setconsole
│ │ -./bin/rx
│ │ -./bin/i2cdump
│ │ -./bin/killall5
│ │ -./bin/lzop
│ │ -./bin/tc
│ │ -./bin/uevent
│ │ -./bin/mim
│ │ -./bin/dd
│ │ -./bin/ftpd
│ │ -./bin/ed
│ │ -./bin/bc
│ │ -./bin/cp
│ │ -./bin/rmmod
│ │ -./bin/route
│ │ -./bin/whois
│ │ -./bin/remove-shell
│ │ -./bin/chpst
│ │ -./bin/tsort
│ │ -./bin/ascii
│ │ -./bin/nsenter
│ │ -./bin/addgroup
│ │ -./bin/sh
│ │ -./bin/dos2unix
│ │ -./bin/basename
│ │ -./bin/ubidetach
│ │ -./bin/nice
│ │ -./bin/tr
│ │ -./bin/mkfs.vfat
│ │ -./bin/deallocvt
│ │ -./bin/svok
│ │ -./bin/patch
│ │ -./bin/svc
│ │ -./bin/ubimkvol
│ │ -./bin/ifconfig
│ │ -./bin/raidautorun
│ │ -./bin/umount
│ │ -./bin/reboot
│ │ -./bin/telnetd
│ │ -./bin/wc
│ │ -./bin/truncate
│ │ -./bin/printf
│ │ -./bin/makedevs
│ │ -./bin/sulogin
│ │ -./bin/mountpoint
│ │ -./bin/devmem
│ │ -./bin/chmod
│ │ -./bin/echo
│ │ -./bin/mke2fs
│ │ -./bin/nc
│ │ -./bin/chown
│ │ -./bin/fakeidentd
│ │ -./bin/linuxrc
│ │ -./bin/sv
│ │ -./bin/chrt
│ │ -./bin/nmeter
│ │ -./bin/telnet
│ │ -./bin/readprofile
│ │ -./bin/freeramdisk
│ │ -./bin/top
│ │ -./bin/gzip
│ │ -./bin/tail
│ │ -./bin/renice
│ │ -./bin/dpkg-deb
│ │ -./bin/crc32
│ │ -./bin/readahead
│ │ -./bin/depmod
│ │ -./bin/shred
│ │ -./bin/vconfig
│ │ -./bin/lpd
│ │ -./bin/lpr
│ │ -./bin/ls
│ │ -./bin/mkfifo
│ │ -./bin/sha1sum
│ │ -./bin/base32
│ │ -./bin/halt
│ │ -./bin/cmp
│ │ -./bin/mkdir
│ │ -./bin/fsfreeze
│ │ -./bin/setserial
│ │ -./bin/i2cget
│ │ -./bin/timeout
│ │ -./bin/vlock
│ │ -./bin/acpid
│ │ -./bin/[[
│ │ -./bin/rmdir
│ │ -./bin/nameif
│ │ -./bin/seedrng
│ │ -./bin/pwdx
│ │ -./bin/ubirename
│ │ -./bin/fuser
│ │ -./bin/brctl
│ │ -./bin/sha256sum
│ │ -./bin/lpq
│ │ -./bin/shuf
│ │ -./bin/ionice
│ │ -./bin/rdate
│ │ -./bin/blockdev
│ │ -./bin/lspci
│ │ -./bin/lsusb
│ │ -./bin/mkfs.ext2
│ │ -./bin/pgrep
│ │ -./bin/mknod
│ │ -./bin/swapoff
│ │ -./bin/ubiattach
│ │ -./bin/udhcpc
│ │ -./bin/nandwrite
│ │ -./bin/mktemp
│ │ -./bin/fsync
│ │ -./bin/time
│ │ -./bin/ifplugd
│ │ -./bin/setfont
│ │ -./bin/fbsplash
│ │ -./bin/fgrep
│ │ -./bin/arch
│ │ -./bin/conspy
│ │ -./bin/expr
│ │ -./bin/false
│ │ -./bin/bzip2
│ │ -./bin/mkfs.minix
│ │ -./bin/cpio
│ │ -./bin/last
│ │ -./bin/sysctl
│ │ -./bin/nohup
│ │ -./bin/add-shell
│ │ -./bin/sort
│ │ -./bin/unlink
│ │ -./bin/lzma
│ │ -./bin/svlogd
│ │ -./bin/wget
│ │ -./bin/seq
│ │ -./bin/unshare
│ │ -./bin/init
│ │ -./bin/hwclock
│ │ -./bin/ubirmvol
│ │ -./bin/grep
│ │ -./bin/hostname
│ │ -./bin/chgrp
│ │ -./bin/modprobe
│ │ -./bin/pstree
│ │ -./bin/mesg
│ │ -./bin/install
│ │ -./bin/[
│ │ -./bin/ping
│ │ -./bin/scriptreplay
│ │ -./bin/dnsdomainname
│ │ -./bin/split
│ │ -./bin/runlevel
│ │ -./bin/setarch
│ │ -./bin/fdflush
│ │ -./bin/realpath
│ │ -./bin/kill
│ │ -./bin/xz
│ │ -./bin/lsattr
│ │ -./bin/cryptpw
│ │ -./bin/dumpleases
│ │ -./bin/clear
│ │ -./bin/ipcalc
│ │ -./bin/which
│ │ -./bin/base64
│ │ -./bin/uniq
│ │ -./bin/ipcs
│ │ -./bin/hd
│ │ -./bin/hostid
│ │ -./bin/unzip
│ │ -./bin/adjtimex
│ │ -./bin/inetd
│ │ -./bin/hexdump
│ │ -./bin/tty
│ │ -./bin/klogd
│ │ -./bin/dhcprelay
│ │ -./bin/pivot_root
│ │ -./bin/setpriv
│ │ -./bin/showkey
│ │ -./bin/nslookup
│ │ -./bin/udpsvd
│ │ -./bin/fdisk
│ │ -./bin/stty
│ │ -./bin/users
│ │ -./bin/iproute
│ │ -./bin/rm
│ │ -./bin/more
│ │ -./bin/tftp
│ │ -./bin/egrep
│ │ -./bin/fsck
│ │ -./bin/insmod
│ │ -./bin/less
│ │ -./bin/su
│ │ -./bin/env
│ │ -./bin/iplink
│ │ -./bin/runsvdir
│ │ -./bin/ip
│ │ -./bin/ftpput
│ │ -./bin/reformime
│ │ -./bin/run-init
│ │ -./bin/sleep
│ │ -./bin/nanddump
│ │ -./bin/who
│ │ -./bin/swapon
│ │ -./bin/iostat
│ │ -./bin/comm
│ │ -./bin/diff
│ │ -./bin/deluser
│ │ -./bin/setuidgid
│ │ -./bin/pwd
│ │ -./bin/test
│ │ -./bin/ifdown
│ │ -./bin/udhcpc6
│ │ -./bin/pipe_progress
│ │ -./bin/ttysize
│ │ -./bin/dmesg
│ │ -./bin/id
│ │ -./bin/chroot
│ │ -./bin/fallocate
│ │ -./bin/powertop
│ │ -./bin/traceroute6
│ │ -./bin/ubirsvol
│ │ -./bin/chvt
│ │ -./bin/nproc
│ │ -./bin/cksum
│ │ -./bin/arping
│ │ -./bin/udhcpd
│ │ -./bin/ifenslave
│ │ -./bin/tac
│ │ -./bin/ps
│ │ -./bin/setfattr
│ │ -./bin/crontab
│ │ -./bin/strings
│ │ -./bin/find
│ │ -./bin/nbd-client
│ │ -./bin/poweroff
│ │ -./bin/bunzip2
│ │ -./bin/ether-wake
│ │ -./bin/crond
│ │ -./bin/dirname
│ │ -./bin/getty
│ │ -./bin/touch
│ │ -./bin/ssl_client
│ │ -./bin/cut
│ │ -./bin/unlzma
│ │ -./bin/blkid
│ │ -./bin/chpasswd
│ │ -./bin/nologin
│ │ -./bin/zcip
│ │ -./bin/uuencode
│ │ -./bin/bzcat
│ │ -./bin/man
│ │ -./bin/iprule
│ │ -./bin/pmap
│ │ -./bin/stat
│ │ -./bin/ping6
│ │ -./bin/modinfo
│ │ -./bin/losetup
│ │ -./bin/gunzip
│ │ -./bin/script
│ │ -./bin/ts
│ │ -./bin/hdparm
│ │ -./bin/tar
│ │ -./bin/delgroup
│ │ -./bin/softlimit
│ │ -./bin/lsof
│ │ -./bin/bootchartd
│ │ -./bin/expand
│ │ -./bin/i2ctransfer
│ │ -./bin/uname
│ │ -./bin/loadkmap
│ │ -./bin/cat
│ │ -./bin/mkswap
│ │ -./bin/popmaildir
│ │ -./bin/ftpget
│ │ -./bin/chat
│ │ -./bin/setkeycodes
│ │ -./bin/fsck.minix
│ │ -./bin/syslogd
│ │ -./bin/makemime
│ │ -./bin/mt
│ │ -./bin/getconf
│ │ -./bin/ubiupdatevol
│ │ -./bin/cal
│ │ -./bin/rtcwake
│ │ -./bin/mkdosfs
│ │ -./bin/rpm2cpio
│ │ -./bin/start-stop-daemon
│ │ -./bin/setlogcons
│ │ -./bin/blkdiscard
│ │ -./bin/watchdog
│ │ -./bin/i2cset
│ │ -./bin/httpd
│ │ -./bin/sha3sum
│ │ -./bin/hexedit
│ │ -./bin/eject
│ │ -./bin/pscan
│ │ -./bin/uptime
│ │ -./bin/true
│ │ -./bin/ar
│ │ -./bin/free
│ │ -./bin/taskset
│ │ -./bin/arp
│ │ -./bin/mpstat
│ │ -./bin/dumpkmap
│ │ -./bin/ln
│ │ -./bin/rdev
│ │ -./bin/w
│ │ -./bin/ntpd
│ │ -./bin/watch
│ │ -./bin/logread
│ │ -./bin/dnsd
│ │ -./bin/tftpd
│ │ -./bin/dpkg
│ │ -./bin/tee
│ │ -./bin/usleep
│ │ -./bin/switch_root
│ │ -./bin/smemcap
│ │ -./bin/head
│ │ -./bin/sed
│ │ -./bin/sync
│ │ -./bin/slattach
│ │ -./bin/openvt
│ │ -./bin/pkill
│ │ -./bin/resize
│ │ -./bin/i2cdetect
│ │ -./bin/reset
│ │ -./bin/flock
│ │ -./bin/envdir
│ │ -./bin/sha512sum
│ │ -./bin/findfs
│ │ -./bin/getopt
│ │ -./bin/killall
│ │ -./bin/logname
│ │ -./bin/loadfont
│ │ -./bin/awk
│ │ -./bin/xxd
│ │ -./bin/run-parts
│ │ -./bin/lsmod
│ │ -./bin/unxz
│ │ -./bin/date
│ │ -./bin/du
│ │ -./bin/microcom
│ │ -./bin/ipaddr
│ │ -./bin/chattr
│ │ -./bin/envuidgid
│ │ -./bin/printenv
│ │ -./bin/hush
│ │ -./bin/md5sum
│ │ -./bin/sum
│ │ -./bin/zcat
│ │ -./bin/od
│ │ -./bin/yes
│ │ -./bin/traceroute
│ │ -./bin/mkpasswd
│ │ -./bin/rev
│ │ -./bin/linux64
│ │ -./bin/dc
│ │ -./bin/fdformat
│ │ -./bin/resume
│ │ -./bin/df
│ │ -./bin/unexpand
│ │ -./bin/linux32
│ │ -./bin/fold
│ │ -./bin/lsscsi
│ │ -./bin/readlink
│ │ -./bin/paste
│ │ -./bin/sendmail
│ │ -./bin/whoami
│ │ -./bin/factor
│ │ -./bin/ifup
│ │ -./bin/partprobe
│ │ -./bin/pidof
│ │ -./bin/beep
│ │ -./bin/ipneigh
│ │ -./bin/adduser
│ │ -./bin/setsid
│ │ -./bin/fgconsole
│ │ -./bin/wall
│ │ -./bin/busybox
│ │ -./bin/fbset
│ │ -./bin/xargs
│ │ -./bin/vi
│ │ -./bin/logger
│ │ -./bin/netstat
│ │ -./bin/mdev
│ │ -./bin/unix2dos
│ │ -./bin/tcpsvd
│ │ -./bin/groups
│ │ -./bin/link
│ │ -./bin/lzcat
│ │ -./bin/passwd
│ │ -./bin/kbd_mode
│ │ -./bin/tree
│ │ -./bin/login
│ │ -./bin/cttyhack
│ │ -./bin/tunctl
│ │ -./bin/volname
│ │ -./bin/nl
│ │ -./bin/uudecode
│ │ -./bin/fatattr
│ │ -./bin/rpm
│ │ -./bin/ash
│ │ -./bin/ipcrm
│ │ -./etc/
│ │ -./etc/shadow
│ │ -./etc/group
│ │ -./etc/network/
│ │ -./etc/network/if-up.d/
│ │ -./etc/network/if-pre-up.d/
│ │ -./etc/network/if-post-down.d/
│ │ -./etc/network/if-down.d/
│ │ -./etc/passwd
│ │ -./etc/localtime
│ │ -./home/
│ │ -./usr/
│ │ -./usr/bin/
│ │ -./usr/bin/env -> ../../bin/env
│ │ -./usr/sbin/
│ │ -./var/
│ │ -./var/www/
│ │ -./var/spool/
│ │ -./var/spool/mail/
│ │ -./tmp/
│ │ -./root/
│ │ -./dev/
│ │ +bin/
│ │ +bin/[
│ │ +bin/[[
│ │ +bin/acpid
│ │ +bin/add-shell
│ │ +bin/addgroup
│ │ +bin/adduser
│ │ +bin/adjtimex
│ │ +bin/ar
│ │ +bin/arch
│ │ +bin/arp
│ │ +bin/arping
│ │ +bin/ascii
│ │ +bin/ash
│ │ +bin/awk
│ │ +bin/base32
│ │ +bin/base64
│ │ +bin/basename
│ │ +bin/bc
│ │ +bin/beep
│ │ +bin/blkdiscard
│ │ +bin/blkid
│ │ +bin/blockdev
│ │ +bin/bootchartd
│ │ +bin/brctl
│ │ +bin/bunzip2
│ │ +bin/busybox
│ │ +bin/bzcat
│ │ +bin/bzip2
│ │ +bin/cal
│ │ +bin/cat
│ │ +bin/chat
│ │ +bin/chattr
│ │ +bin/chgrp
│ │ +bin/chmod
│ │ +bin/chown
│ │ +bin/chpasswd
│ │ +bin/chpst
│ │ +bin/chroot
│ │ +bin/chrt
│ │ +bin/chvt
│ │ +bin/cksum
│ │ +bin/clear
│ │ +bin/cmp
│ │ +bin/comm
│ │ +bin/conspy
│ │ +bin/cp
│ │ +bin/cpio
│ │ +bin/crc32
│ │ +bin/crond
│ │ +bin/crontab
│ │ +bin/cryptpw
│ │ +bin/cttyhack
│ │ +bin/cut
│ │ +bin/date
│ │ +bin/dc
│ │ +bin/dd
│ │ +bin/deallocvt
│ │ +bin/delgroup
│ │ +bin/deluser
│ │ +bin/depmod
│ │ +bin/devmem
│ │ +bin/df
│ │ +bin/dhcprelay
│ │ +bin/diff
│ │ +bin/dirname
│ │ +bin/dmesg
│ │ +bin/dnsd
│ │ +bin/dnsdomainname
│ │ +bin/dos2unix
│ │ +bin/dpkg
│ │ +bin/dpkg-deb
│ │ +bin/du
│ │ +bin/dumpkmap
│ │ +bin/dumpleases
│ │ +bin/echo
│ │ +bin/ed
│ │ +bin/egrep
│ │ +bin/eject
│ │ +bin/env
│ │ +bin/envdir
│ │ +bin/envuidgid
│ │ +bin/ether-wake
│ │ +bin/expand
│ │ +bin/expr
│ │ +bin/factor
│ │ +bin/fakeidentd
│ │ +bin/fallocate
│ │ +bin/false
│ │ +bin/fatattr
│ │ +bin/fbset
│ │ +bin/fbsplash
│ │ +bin/fdflush
│ │ +bin/fdformat
│ │ +bin/fdisk
│ │ +bin/fgconsole
│ │ +bin/fgrep
│ │ +bin/find
│ │ +bin/findfs
│ │ +bin/flock
│ │ +bin/fold
│ │ +bin/free
│ │ +bin/freeramdisk
│ │ +bin/fsck
│ │ +bin/fsck.minix
│ │ +bin/fsfreeze
│ │ +bin/fstrim
│ │ +bin/fsync
│ │ +bin/ftpd
│ │ +bin/ftpget
│ │ +bin/ftpput
│ │ +bin/fuser
│ │ +bin/getconf
│ │ +bin/getopt
│ │ +bin/getty
│ │ +bin/grep
│ │ +bin/groups
│ │ +bin/gunzip
│ │ +bin/gzip
│ │ +bin/halt
│ │ +bin/hd
│ │ +bin/hdparm
│ │ +bin/head
│ │ +bin/hexdump
│ │ +bin/hexedit
│ │ +bin/hostid
│ │ +bin/hostname
│ │ +bin/httpd
│ │ +bin/hush
│ │ +bin/hwclock
│ │ +bin/i2cdetect
│ │ +bin/i2cdump
│ │ +bin/i2cget
│ │ +bin/i2cset
│ │ +bin/i2ctransfer
│ │ +bin/id
│ │ +bin/ifconfig
│ │ +bin/ifdown
│ │ +bin/ifenslave
│ │ +bin/ifplugd
│ │ +bin/ifup
│ │ +bin/inetd
│ │ +bin/init
│ │ +bin/insmod
│ │ +bin/install
│ │ +bin/ionice
│ │ +bin/iostat
│ │ +bin/ip
│ │ +bin/ipaddr
│ │ +bin/ipcalc
│ │ +bin/ipcrm
│ │ +bin/ipcs
│ │ +bin/iplink
│ │ +bin/ipneigh
│ │ +bin/iproute
│ │ +bin/iprule
│ │ +bin/iptunnel
│ │ +bin/kbd_mode
│ │ +bin/kill
│ │ +bin/killall
│ │ +bin/killall5
│ │ +bin/klogd
│ │ +bin/last
│ │ +bin/less
│ │ +bin/link
│ │ +bin/linux32
│ │ +bin/linux64
│ │ +bin/linuxrc
│ │ +bin/ln
│ │ +bin/loadfont
│ │ +bin/loadkmap
│ │ +bin/logger
│ │ +bin/login
│ │ +bin/logname
│ │ +bin/logread
│ │ +bin/losetup
│ │ +bin/lpd
│ │ +bin/lpq
│ │ +bin/lpr
│ │ +bin/ls
│ │ +bin/lsattr
│ │ +bin/lsmod
│ │ +bin/lsof
│ │ +bin/lspci
│ │ +bin/lsscsi
│ │ +bin/lsusb
│ │ +bin/lzcat
│ │ +bin/lzma
│ │ +bin/lzop
│ │ +bin/makedevs
│ │ +bin/makemime
│ │ +bin/man
│ │ +bin/md5sum
│ │ +bin/mdev
│ │ +bin/mesg
│ │ +bin/microcom
│ │ +bin/mim
│ │ +bin/mkdir
│ │ +bin/mkdosfs
│ │ +bin/mke2fs
│ │ +bin/mkfifo
│ │ +bin/mkfs.ext2
│ │ +bin/mkfs.minix
│ │ +bin/mkfs.vfat
│ │ +bin/mknod
│ │ +bin/mkpasswd
│ │ +bin/mkswap
│ │ +bin/mktemp
│ │ +bin/modinfo
│ │ +bin/modprobe
│ │ +bin/more
│ │ +bin/mount
│ │ +bin/mountpoint
│ │ +bin/mpstat
│ │ +bin/mt
│ │ +bin/mv
│ │ +bin/nameif
│ │ +bin/nanddump
│ │ +bin/nandwrite
│ │ +bin/nbd-client
│ │ +bin/nc
│ │ +bin/netstat
│ │ +bin/nice
│ │ +bin/nl
│ │ +bin/nmeter
│ │ +bin/nohup
│ │ +bin/nologin
│ │ +bin/nproc
│ │ +bin/nsenter
│ │ +bin/nslookup
│ │ +bin/ntpd
│ │ +bin/od
│ │ +bin/openvt
│ │ +bin/partprobe
│ │ +bin/passwd
│ │ +bin/paste
│ │ +bin/patch
│ │ +bin/pgrep
│ │ +bin/pidof
│ │ +bin/ping
│ │ +bin/ping6
│ │ +bin/pipe_progress
│ │ +bin/pivot_root
│ │ +bin/pkill
│ │ +bin/pmap
│ │ +bin/popmaildir
│ │ +bin/poweroff
│ │ +bin/powertop
│ │ +bin/printenv
│ │ +bin/printf
│ │ +bin/ps
│ │ +bin/pscan
│ │ +bin/pstree
│ │ +bin/pwd
│ │ +bin/pwdx
│ │ +bin/raidautorun
│ │ +bin/rdate
│ │ +bin/rdev
│ │ +bin/readahead
│ │ +bin/readlink
│ │ +bin/readprofile
│ │ +bin/realpath
│ │ +bin/reboot
│ │ +bin/reformime
│ │ +bin/remove-shell
│ │ +bin/renice
│ │ +bin/reset
│ │ +bin/resize
│ │ +bin/resume
│ │ +bin/rev
│ │ +bin/rm
│ │ +bin/rmdir
│ │ +bin/rmmod
│ │ +bin/route
│ │ +bin/rpm
│ │ +bin/rpm2cpio
│ │ +bin/rtcwake
│ │ +bin/run-init
│ │ +bin/run-parts
│ │ +bin/runlevel
│ │ +bin/runsv
│ │ +bin/runsvdir
│ │ +bin/rx
│ │ +bin/script
│ │ +bin/scriptreplay
│ │ +bin/sed
│ │ +bin/seedrng
│ │ +bin/sendmail
│ │ +bin/seq
│ │ +bin/setarch
│ │ +bin/setconsole
│ │ +bin/setfattr
│ │ +bin/setfont
│ │ +bin/setkeycodes
│ │ +bin/setlogcons
│ │ +bin/setpriv
│ │ +bin/setserial
│ │ +bin/setsid
│ │ +bin/setuidgid
│ │ +bin/sh
│ │ +bin/sha1sum
│ │ +bin/sha256sum
│ │ +bin/sha3sum
│ │ +bin/sha512sum
│ │ +bin/showkey
│ │ +bin/shred
│ │ +bin/shuf
│ │ +bin/slattach
│ │ +bin/sleep
│ │ +bin/smemcap
│ │ +bin/softlimit
│ │ +bin/sort
│ │ +bin/split
│ │ +bin/ssl_client
│ │ +bin/start-stop-daemon
│ │ +bin/stat
│ │ +bin/strings
│ │ +bin/stty
│ │ +bin/su
│ │ +bin/sulogin
│ │ +bin/sum
│ │ +bin/sv
│ │ +bin/svc
│ │ +bin/svlogd
│ │ +bin/svok
│ │ +bin/swapoff
│ │ +bin/swapon
│ │ +bin/switch_root
│ │ +bin/sync
│ │ +bin/sysctl
│ │ +bin/syslogd
│ │ +bin/tac
│ │ +bin/tail
│ │ +bin/tar
│ │ +bin/taskset
│ │ +bin/tc
│ │ +bin/tcpsvd
│ │ +bin/tee
│ │ +bin/telnet
│ │ +bin/telnetd
│ │ +bin/test
│ │ +bin/tftp
│ │ +bin/tftpd
│ │ +bin/time
│ │ +bin/timeout
│ │ +bin/top
│ │ +bin/touch
│ │ +bin/tr
│ │ +bin/traceroute
│ │ +bin/traceroute6
│ │ +bin/tree
│ │ +bin/true
│ │ +bin/truncate
│ │ +bin/ts
│ │ +bin/tsort
│ │ +bin/tty
│ │ +bin/ttysize
│ │ +bin/tunctl
│ │ +bin/ubiattach
│ │ +bin/ubidetach
│ │ +bin/ubimkvol
│ │ +bin/ubirename
│ │ +bin/ubirmvol
│ │ +bin/ubirsvol
│ │ +bin/ubiupdatevol
│ │ +bin/udhcpc
│ │ +bin/udhcpc6
│ │ +bin/udhcpd
│ │ +bin/udpsvd
│ │ +bin/uevent
│ │ +bin/umount
│ │ +bin/uname
│ │ +bin/unexpand
│ │ +bin/uniq
│ │ +bin/unix2dos
│ │ +bin/unlink
│ │ +bin/unlzma
│ │ +bin/unshare
│ │ +bin/unxz
│ │ +bin/unzip
│ │ +bin/uptime
│ │ +bin/users
│ │ +bin/usleep
│ │ +bin/uudecode
│ │ +bin/uuencode
│ │ +bin/vconfig
│ │ +bin/vi
│ │ +bin/vlock
│ │ +bin/volname
│ │ +bin/w
│ │ +bin/wall
│ │ +bin/watch
│ │ +bin/watchdog
│ │ +bin/wc
│ │ +bin/wget
│ │ +bin/which
│ │ +bin/who
│ │ +bin/whoami
│ │ +bin/whois
│ │ +bin/xargs
│ │ +bin/xxd
│ │ +bin/xz
│ │ +bin/xzcat
│ │ +bin/yes
│ │ +bin/zcat
│ │ +bin/zcip
│ │ +dev/
│ │ +etc/
│ │ +etc/group
│ │ +etc/localtime
│ │ +etc/network/
│ │ +etc/network/if-down.d/
│ │ +etc/network/if-post-down.d/
│ │ +etc/network/if-pre-up.d/
│ │ +etc/network/if-up.d/
│ │ +etc/passwd
│ │ +etc/shadow
│ │ +home/
│ │ +root/
│ │ +tmp/
│ │ +usr/
│ │ +usr/bin/
│ │ +usr/bin/env -> ../../bin/env
│ │ +usr/sbin/
│ │ +var/
│ │ +var/spool/
│ │ +var/spool/mail/
│ │ +var/www/
```

</details>

(entirely superficial :rocket:)

**Review by yosifkit — APPROVED** [review]

**Pull request merged**